src/analyzing/AST_to_IL.ml

```diff
@@ -445,8 +453,14 @@ and assign env lhs tok rhs_exp e_gen =
 and expr_aux env ?(void = false) e_gen =
```

Comment:
could you explain what this void parameter is?

Reply:
Will add a comment.